test/unit/volume_manager_btrfs_test.py

```diff
@@ -6,7 +6,10 @@
 from textwrap import dedent
 import datetime
 
-from .test_helper import *
+from .test_helper import (
+    raises, patch_open
+)
```

Comment:
@schaefi I am not klugscheissing, but when I do...
The `(` and `)` parenthesis in imports are _only_ needed for a [multi-line imports](https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses). In this case you are doing an overkill. It is enough:
```python
from .test_helper import raises, patch_open
```